### PR TITLE
Add docs for quickstart with raw manifests on README

### PR DESCRIPTION
**Pull request by ffjlabo**
**What this PR does / why we need it**:

as title

**Which issue(s) this PR fixes**:

Part of https://github.com/pipe-cd/pipecd/issues/5148

**Does this PR introduce a user-facing change?**:

- **How are users affected by this change**:
- **Is this breaking change**:
- **How to migrate (if breaking change)**:

### CODE REVIEW

File: quickstart/README.md [hunk 1]
```diff
@@ -3,6 +3,39 @@
...
+kubectl create namespace pipecd
+```
+
+2. deploy Control Plane to the namespace: pipecd
+
+```
+kubectl apply -n pipecd -f ./manifests/control-plane.yaml
+```
+
+3. create piped on the Web UI
```
Comment (Warashi) [inline quickstart/README.md:21]: I think it's nice for users to show kubectl port-forward command and URL of the Web UI.
WDYT?

Comment (ffjlabo) [reply quickstart/README.md:21]: @Warashi 
It's a nice idea. But I wonder if I should do so 🤔 
For me, this doc is like a memo to describe whats the manifest is. 
Also there is the actual process of the quickstart on the docs. https://pipecd.dev/docs-v0.48.x/quickstart/

Comment (Warashi) [reply quickstart/README.md:21]: OK, I got it. I agree that we have other documents about these manifests and don't have to show detailed commands here.

**Review by khanhtc1202 — APPROVED** [review]
Thank you 👍

**Pull request merged**